packages/cli/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+- Fix: Removed the `--` from the `safe-replace` option's alias. [#2423] (https://github.com/zowe/zowe-cli/pull/2423)
```

Comment:
Since this was not published, we can remove this changelog entry and label the PR as no-changelog. 😋

Reply:
Thanks for removing! 🙏 
no need to worry about the Changelog check 🙏 
https://github.com/zowe/zowe-cli/actions/runs/13055703586/job/36426127073?pr=2423